### DISCUSSION TITLE
I can not setFenodes with https

### DISCUSSION THREAD

**Discussion by ducthangbui** [apache/doris#27692] [A - General / Q&A] [answered]
I run FE with https on port 443 but when I use dorisBuilder.setFenodes, I receive error log No Doris FE is available

![image](https://github.com/apache/doris/assets/24984239/2567c860-f887-4fe2-b07a-d5486e49aaee)

![image](https://github.com/apache/doris/assets/24984239/3e3747a7-8446-4aaa-9d3b-9c5d38e15af8)

**Comment by LemonLiTree** [comment, marked as answer]
1. Confirm that fe is alive
2. Check configuration

**Reply by ducthangbui** [reply]
1. FE is alive
2. FE is running on port 443 in my config file
